common/db_test.go

```diff
@@ -656,6 +659,83 @@ func TestLoadWinningTickets_GivenTwoSessionsWithTickets_ReturnsAllTickets(t *tes
 	assert.Equal(recipientRand1, recipientRands[1])
 }
 
+func TestInsertMiniHeader_ReturnsFindLatestMiniHeader(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	assert := assert.New(t)
+	require := require.New(t)
+	require.Nil(err)
+
+	h0 := defaultMiniHeader(t)
+	err = dbh.InsertMiniHeader(h0)
+	require.Nil(err)
+
+	h0db, err := dbh.FindLatestMiniHeader()
```

Comment:
Test the error paths for `FindLatestMiniHeader()`?

Reply:
This is being checked in the Delete test, where after there are no more remaining entries `FindLatestMiniHeader` returns an error. I have added a comment there for clarity.

I didn't add a seperate test for `FindLatestMiniHeader` because its functionality is being tested in the other tests and it's correct working is crucial for those tests to pass.